lifecycle_test.go

```diff
@@ -300,6 +300,39 @@ func TestPreCreateModifierHook(t *testing.T) {
 			"Networking config's network ID should be retrieved from Docker",
 		)
 	})
+	
+	t.Run("Request contains exposed port modifiers", func(t *testing.T) {
+		// reqWithModifiers {
+		req := ContainerRequest{
+			Image: nginxAlpineImage, // alpine image does expose port 80
+			HostConfigModifier: func(hostConfig *container.HostConfig) {
+				hostConfig.PortBindings = nat.PortMap{
+					"80/tcp": []nat.PortBinding{
+						{
+							HostIP:   "localhost",
+							HostPort: "8080",
+						},
+					},
+				}
+			},
+			ExposedPorts: []string{"80"},
+		}
+		// }
```

Comment:
Let's remove this comment, which is used by the docs page to render a code snippet by name
```suggestion
```